RecoTracker/DisplacedRegionalTracking/plugins/DisplacedRegionProducer.h

```diff
@@ -0,0 +1,64 @@
+#ifndef RecoTracker_DisplacedRegionalTracking_DisplacedRegionProducer_h
+#define RecoTracker_DisplacedRegionalTracking_DisplacedRegionProducer_h
+
+#include <list>
+#include <vector>
+#include <limits>
+#include <string>
+#include <atomic>
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/Candidate/interface/VertexCompositeCandidate.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/Math/interface/Vector3D.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+
+#include "RecoTracker/DisplacedRegionalTracking/interface/DisplacedRegion.h"
+
+using namespace std;
```

Comment:
CMS code rule violation

Rule1 : https://raw.githubusercontent.com/cms-sw/cmssw/master/Utilities/ReleaseScripts/python/cmsCodeRules/config.py
Search for "using namespace" or "using std::" in header files

/RecoTracker/DisplacedRegionalTracking/plugins/DisplacedRegionProducer.h
[30]

Reply:
I have moved `using namespace std;` out of the header file.